Exercises/NoModules/Chapter 10/Soln10_03.cpp

```diff
@@ -0,0 +1,47 @@
+// Exercise 10-3.cpp
+// Defining a function template for adding numbers, 
+// and an overload that works for pointer types.
+// Extra: also make plus() work with string literals...
+#include <iostream>
+#include <string>
+#include <string_view>
+
+template <typename T>
+T plus(const T& a, const T& b)
+{
+    return a + b;
+}
+
+// Overload with another template for pointer types
+template <typename T>
+T plus(const T* a, const T* b)
+{
+    return *a + *b;
+}
+
+// Result cannot be const char* (could at best be pointer to local variable),
```

Comment:
I would remove "(could at best be pointer to local variable),"
readers might think that that's ok sometimes. but it never is.